Composer/packages/lib/code-editor/src/lg/useDebounce.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+
+export const useDebounce = <T>(value: T, delay: number) => {
```

Comment:
Nit: useDebounce is useful to many component, consider moving it up (bfc/shared?)